packages/@glimmer/debug/lib/dism/opcode.ts

```diff
@@ -29,7 +29,7 @@ export function describeOp(
   program: Program,
   meta: Nullable<BlockMetadata>
 ): Fragment {
-  const { name, params } = debugOp(program, op, meta)!;
+  const { name, params } = debugOp(program, op, meta);
```

Comment:
There's a decent amount of totally unnecessary `!` assertions, and they were removed via autofix.